src/sentry/integrations/notify_disable.py

```diff
@@ -39,13 +39,14 @@ def notify_disable(
     organization: Organization,
     integration_name: str,
     redis_key: str,
+    integration_slug: Union[str, None] = None,
     project: Union[str, None] = None,
 ):
 
     integration_link = get_url(
         organization,
         get_provider_type(redis_key),
-        integration_name,
+        str(integration_slug) if "sentry-app" in redis_key else integration_name,
```

Comment:
Is this not already a string?

Reply:
Backend typing said ` "get_url" has  incompatible type "Optional[str]"; expected "str"` so that's why I called `str()` here

Reply:
Ah ok you can change it to:

`integration_slug if "sentry-app" in redis_key and integration_slug else integration_name`

since `integration_slug` is optional